src/lib.rs

```diff
@@ -365,6 +365,27 @@ where
     ncs: NCS,
 }
 
+/// Calculate the Config1::DataRate from a given sasmple rate
+pub fn datarate_from_samplerate(sample_rate: f32, om: OperatingMode) -> Result<DataRate,()> {
```

Comment:
```suggestion
pub fn datarate_from_samplerate(sample_rate: f32, operationg_mode: OperatingMode) -> Result<DataRate,()> {
```

Reply:
This argument is relevant to documentation & user comprehension, which is why I think the full name is relevant :)

Reply:
On second thought, while I still feel the same with abbreviations in general, I think it's not so critical as the type already tells what's going on :)